site/app/libraries/PollUtils.php

```diff
@@ -23,7 +23,7 @@ public static function getPollExportData(array $polls): array {
                 'responses' => [],
                 'correct_responses' => [],
                 'duration' => $poll->getDuration()->format('P%yY%mM%dDT%hH%iM%sS'),
-                'end_time' => $poll->getEndTime()->format('Y-m-d'),
+                'end_time' => $poll->getEndTime() === null ? $poll->getEndTime() : $poll->getEndTime()->format('Y-m-d'),
```

Comment:
Isn't this rather silly?  If `$poll->getEndTime()` returns null, why call it again?

Perhaps something like this would be better:
```suggestion
                'end_time' => $poll->getEndTime()?->format('Y-m-d'),
```

Reply:
ahh, didn't know you could do that applied this change